data/test/vtexplain/unsharded-output.json

```diff
@@ -82,11 +81,11 @@
                     },
                     {
                         "Time": 1,
-                        "SQL": "insert into t1(id, intval, floatval) values (1, 2, 3.14)"
+                        "SQL": "commit"
```

Comment:
This output change isn't actually a correct reflection of the execution order, since the misguided attempt to improve test stability by sorting queries in the same time cycle had the downside of making this no longer reflect the logical order of statements.

This, among other things, is fixed in #3532 so I think it would be best to merge that first, then re-run the vtexplain tests to generate what should hopefully be a more logical output diff in which all I would expect here is the time to change from 2 to 1.